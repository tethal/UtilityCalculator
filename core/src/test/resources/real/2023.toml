[general]
start_date = 2023-02-15
end_date = 2023-12-31
tenant = ["Marie Černá", "Jindřišská 16", "111 50 Praha 1"]
owner = ["Jan Novák", "majitel@example.com"]
report_place = "V Praze"
report_date = 2024-04-27
sources = ["Vyúčtování služeb od Společenství vlastníků za rok 2023", "Příloha č. 3 k vyhlášce č.269/2015 Sb."]

[[deposits]]
description="únor"
count=1
amount=1500

[[deposits]]
description="březen - prosinec"
count=10
amount=3000

[[heating]]
annual_cost = 10922
start_date = 2023-01-01
end_date = 2023-12-31

[[other_fees]]
start_date = 2023-01-01
end_date = 2023-12-31
annual_cost = 10316

[[cold_water.reading]]
meter_id = "1"
reading_date = 2023-02-15
state = 23.1

[[cold_water.reading]]
meter_id = "1"
reading_date = 2024-01-01
state = 60.7

[[cold_water.tariff]]
start_date = 2023-01-01
end_date = 2023-12-31
unit_amount = 139.6041

[[hot_water.reading]]
meter_id = "1"
reading_date = 2023-02-15
state = 14

[[hot_water.reading]]
meter_id = "1"
reading_date = 2024-01-01
state = 30.7

[[hot_water.tariff]]
start_date = 2023-01-01
end_date = 2023-12-31
unit_amount = 105.3

[[hot_water.heating_basic_cost]]
start_date = 2023-01-01
end_date = 2023-12-31
annual_cost = 2588.88

[[hot_water.heating_consumable_tariff]]
start_date = 2023-01-01
end_date = 2023-12-31
unit_amount = 242.66
