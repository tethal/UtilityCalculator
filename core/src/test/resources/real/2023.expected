Report:
  dateRange: 2023-02-15...2024-01-01
  tenant: Marie Černá, Jindřišská 16, 111 50 Praha 1
  owner: Jan Novák, majitel@example.com
  placeDate: V Praze, 2024-04-27
  sources: Vyúčtování služeb od Společenství vlastníků za rok 2023, Příloha č. 3 k vyhlášce č.269/2015 Sb.
  sections:
    Studená voda: 5249.11
    Teplá voda: 8076.20
    Vytápění: 7973.06
    Ostatní poplatky: 9026.50
    Přijaté zálohy: -31500
    TOTAL: -1175.13

Studená voda:
  readings:
    [1] 2023-02-15...2024-01-01: 23.1...60.7 = 37.6
  priceList:
    2023-02-15...2024-01-01: 37.600 * 139.6041 = 5249.11

Teplá voda:
  readings:
    [1] 2023-02-15...2024-01-01: 14...30.7 = 16.7
  priceList:
    2023-02-15...2024-01-01: 16.700 * 105.3 = 1758.51
  heatingBasicParts:
    2023-02-15...2024-01-01: 10.50 * 215.74 = 2265.27
  heatingConsumableParts:
    2023-02-15...2024-01-01: 16.700 * 242.66 = 4052.42

Vytápění:
  2023-02: 0.50 * 10922 * 0.16 = 873.76
  2023-03: 1.00 * 10922 * 0.14 = 1529.08
  2023-04: 1.00 * 10922 * 0.09 = 982.98
  2023-05: 1.00 * 10922 * 0.02 = 218.44
  2023-06: 1.00 * 10922 * 0.0 = 0.00
  2023-07: 1.00 * 10922 * 0.0 = 0.00
  2023-08: 1.00 * 10922 * 0.0 = 0.00
  2023-09: 1.00 * 10922 * 0.01 = 109.22
  2023-10: 1.00 * 10922 * 0.08 = 873.76
  2023-11: 1.00 * 10922 * 0.14 = 1529.08
  2023-12: 1.00 * 10922 * 0.17 = 1856.74

Ostatní poplatky:
  2023-02-15...2024-01-01: 10.50 * 859.67 = 9026.50

Přijaté zálohy:
  únor: 1 * 1500 = 1500
  březen - prosinec: 10 * 3000 = 30000
